providers/linux/os_test.go

```diff
@@ -15,6 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// +build linux
```

Comment:
I think it fails on Windows because the symlinks in the git repo don't work on Windows. How about `!windows` instead? (I liked being able to test this from darwin.)

Reply:
Sounds good, I'll make the change.